ftests/nuxeo-platform-explorer-ftests/src/test/java/org/nuxeo/functionaltests/explorer/ITExplorerTest.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * (C) Copyright 2014-2020 Nuxeo SA (http://nuxeo.com/) and contributors.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the GNU Lesser General Public License
+ * (LGPL) version 2.1 which accompanies this distribution, and is available at
+ * http://www.gnu.org/licenses/lgpl-2.1.html
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * Contributors:
+ *     Nuxeo
+ *     Anahide Tchertchian
+ */
+package org.nuxeo.functionaltests.explorer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.nuxeo.functionaltests.AbstractTest;
+import org.nuxeo.functionaltests.Locator;
+import org.nuxeo.functionaltests.RestHelper;
+import org.nuxeo.functionaltests.explorer.pages.ArtifactHomePage;
+import org.nuxeo.functionaltests.explorer.pages.ArtifactPage;
+import org.nuxeo.functionaltests.explorer.pages.ExplorerHomePage;
+import org.nuxeo.functionaltests.pages.DocumentBasePage.UserNotConnectedException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Test explorer main webengine pages.
+ *
+ * @since 11.1
+ */
+public class ITExplorerTest extends AbstractTest {
+
+    @Before
+    public void before() {
+        RestHelper.createUser(TEST_USERNAME, TEST_PASSWORD, null, null, null, null, "members");
+    }
+
+    @After
+    public void after() {
+        RestHelper.cleanup();
+    }
+
+    protected void doLogin() {
```

Comment:
You could rely on `@Before` for this one.

Reply:
thanks indeed, although i sometimes log in as admin user instead, so i'll take that option in next PRs (more ftests are coming for admin user for https://jira.nuxeo.com/browse/NXP-20246)

Reply:
Actually i got confused with JSF tests (similar logics, but one additional test checking admin center), will do here :+1: